### PR TITLE
MovieList styled fixing after wrong merge

### PR DESCRIPTION
**Pull request by GrzegorzPacewicz**
This one needs to be changed too.

### CODE REVIEW

File: src/features/people/PeopleListPage/index.js [hunk 1]
```diff
@@ -64,7 +65,7 @@ const PeopleListPage = () => {
                   : `Popular people`
               }
             />
-            <Layout>
+            <PeopleContainer>
```
Comment (greedyChicken) [inline src/features/people/PeopleListPage/index.js:67]: can you remove Layout from imports then?

**Review by greedyChicken — COMMENTED** [review]
check this Layout import please

File: src/features/people/PeopleListPage/index.js (hunk 1 above)
Comment (GrzegorzPacewicz) [reply src/features/people/PeopleListPage/index.js:67]: Done! Thanks!

**Review by greedyChicken — APPROVED** [review]
ok, I think it's all right

**Pull request merged**